### ISSUE TITLE
Release f1dataR 1.2.1.9000

### ISSUE THREAD

**Issue by SCasanova** [SCasanova/f1dataR#137]
First release:

* [x] `usethis::use_cran_comments()`
* [x] Update (aspirational) install instructions in README
* [x] Proofread `Title:` and `Description:`
* [x] Check that all exported functions have `@return` and `@examples`
* [x] Check that `Authors@R:` includes a copyright holder (role 'cph')
* [x] Check [licensing of included files](https://r-pkgs.org/license.html#code-you-bundle)
* [x] Review <https://github.com/DavisVaughan/extrachecks>

Prepare for release:

* [x] `git pull`
* [x] `devtools::build_readme()`
* [x] `urlchecker::url_check()`
* [x] `devtools::check(remote = TRUE, manual = TRUE)`
* [x] `devtools::check_win_devel()`
* [x] `rhub::check_for_cran()`
* [x] `git push`

Submit to CRAN:

* [x] `usethis::use_version('patch')`
* [ ] `devtools::submit_cran()`
* [ ] Approve email

Wait for CRAN...

* [ ] Accepted :tada:
* [ ] `git push`
* [ ] `usethis::use_github_release()`
* [ ] `usethis::use_dev_version()`
* [ ] `git push`

**Comment by pbulsink** [comment]
Possibly the only outstanding item from DavisVaughan/extrachecks is the [\dontrun{} section](https://github.com/DavisVaughan/extrachecks#you-used-dontrun-in-an-example-and-got-a-note-about-that). Should we wrap in interactive() instead?

**Comment by pbulsink** [comment]
https://github.com/marioangst/motifr/issues/40 discusses python failures on rhub, could be worth being more aggressive on the skip tests if no python & use prebuilt vignettes?

**Comment by SCasanova** [comment]
What do you mean by pre-built?

**Comment by pbulsink** [comment]
We can submit as-is, and if they flag build failures because of the vignette building requiring python (for introduction.Rmd) either precompile it as a .pdf or fake the computation via the tricks outlined here: https://blog.r-hub.io/2020/06/03/vignettes/#fnref:3

**Comment by pbulsink** [comment]
We'll likely also get a comment about this, but have to wait and see what they prefer (CRAN doesn't want 301 redirect URLs but also doesn't want http://) 
![Screenshot_20230728-091003](https://github.com/SCasanova/f1dataR/assets/5419974/f4b6cb55-a87a-4f25-8896-c0a215caf886)

I can update the relevant doc to include the trailing slash, and see if that'll be close enough for them.

**Comment by pbulsink** [comment]
> What do you mean by pre-built?

Having looked at it a bit more, I think I prefer this option: https://ropensci.org/blog/2019/12/08/precompute-vignettes/#the-solution-locally-knitting-rmarkdown 

This way the vignette code is still really available, and things are build-able on CRAN/GitHub Actions without as many hits to Ergast.

**Comment by SCasanova** [comment]
Sounds good

**Comment by SCasanova** [comment]
I will submit anyway and see what their comments are

**Comment by SCasanova** [comment]
CRAN is on vacation until the 7th... we have some leeway

**Comment by pbulsink** [comment]
rhub::check_for_cran() now only return notes after #146

**Comment by SCasanova** [comment]
Package has been submitted! I'm ready to be hurt by their comments

**Comment by SCasanova** [comment]
We passed auto-check. Might fail manual check due to typo in README at the time of submission :(

**Comment by SCasanova** [comment]
Superseded by #164
